test/providers/index.test.ts

```diff
@@ -1,13 +1,14 @@
 import td from 'testdouble'
 
 import providers from '../../src/ci_providers'
+import { UploaderInputs } from '../../src/types'
 
 describe('CI Providers', () => {
   afterEach(() => {
     td.reset()
   })
 
-  it('is an array of CI providers', () =>
+  it('check that each provider', () =>
```

Comment:
Nested `it` blocks might be somewhat fragile. I don't believe it's an officially supported pattern.

1. Change the `it` on line 11 to `describe`:
   ```suggestion
     describe('check that each provider', () =>
   ```
2. Move the contents of line 12 into its own `it` block

Reply:
I'm not nesting `it`, am I? 👀